source/install/native.rst

```diff
@@ -99,7 +99,7 @@ Now that you have a QIIME 2 environment, activate it using the environment's nam
 
    conda activate qiime2-2020.5
 
-To deactivate an environment, run ``source deactivate``.
+To deactivate an environment, run ``conda deactivate``.
```

Comment:
Good catch! I updated everything back in https://github.com/qiime2/docs/pull/456 to use `conda activate`, looks like I missed this line.